sonic_platform_base/sfp_base.py

```diff
@@ -262,6 +264,15 @@ def get_power_override(self):
         """
         raise NotImplementedError
 
+    def get_power_set(self):
```

Comment:
I think the "Power set" wording is taken from SFF 8636. What does get_power_set() mean for QSFP-DD or modules that follow CMIS spec?

Reply:
Yes, from what I can tell the original lpmode/power override APIs were designed only with SFF 8436/8636 in mind, and get_power_set would go with them. It doesn't really a have a direct equivalent in CMIS. My interpretation would be that LowPwrRequestSW in CMIS rev 5 (ForceLowPwr in rev 4) essentially behaves like power_set AND power_override, i.e.:
- If LowPwrRequestSW is 1, then power_override is 1 and power_set is 1
- If LowPwrRequestSW is 0, then power_override is 0 and power_set is X
For CMIS we may want to design new APIs for power-related functionality. For now this is sufficient since the only (Arista) platforms we're aware of that don't support the lpmode pin are using 8436/8636-compliant xcvrs and not CMIS-compliant xcvrs.

Reply:
my only concern here is that these APIs will be used by same port that support optics following two different spec and api wording doesn't seem meaning for CMIS. Rest of the (get/set) APIs in sfp_base.py is applicable for almost all optics.  How is get_power_set() going to be used. ? Can't this be absorbed in platform specific sfp.py?